source/documentation/deploying_services/postgresql.md.erb

```diff
@@ -244,6 +246,61 @@ To move data between two PaaS-hosted PostgreSQL databases:
 
     where `DESTINATION_SERVICE_NAME` is the name of the target database.
 
+### Expected Errors
```

Comment:
Make it linkable, but also have it nested under "Import and export bulk data to and from a PostgreSQL database"
```suggestion
<h4 id="import-data-expected-errors">Expected Errors</h4>
```